package.json

```diff
@@ -60,6 +60,7 @@
                         "request": "launch",
                         "platform": "android",
                         "internalDebuggerPort": 9090,
+                        "extensionServerPort": 8099,
```

Comment:
The thing is that we are polluting the UI with things he shouldn't really care about. It would be nice if we could magically solve this problem under the rug.